src/test/e2e/psp.feature.ts

```diff
@@ -0,0 +1,189 @@
+import fs from 'fs';
+import path from 'path';
+import { mockReadFileSync } from '../mock-utils';
+import { ethers } from 'ethers';
+import * as abi from '@api3/airnode-abi';
+import * as node from '@api3/airnode-node';
+import * as psp from '../../handlers/psp';
+import * as config from '../../config';
+import { buildAirnodeConfig, buildAirkeeperConfig, buildLocalConfig } from '../config/config';
+import { PROTOCOL_ID_PSP } from '../../constants';
+
+jest.setTimeout(30_000);
```

Comment:
There's a [bug in Jest >v.27](https://github.com/facebook/jest/issues/11607) where a **setTimeout** doesn't work correctly inside of a describe block :smile:

Reply:
It would be nice to add that as a comment (with a link to that issue).